integration-tests/plugins/.yarnrc.yml

```diff
@@ -0,0 +1,4 @@
+npmRegistryServer: "http://localhost:4873"
```

Comment:
question: Do we need this or is it specific to your local env?

Reply:
It is required to run `medusa-dev` in the `integration-tests/plugins` dir locally; not specific to my env only. 

It is similar to `integration-tests/api/.yarnrc.yml`.